vernac/declareDef.ml

```diff
@@ -69,12 +69,15 @@ let declare_definition ~name ~scope ~kind ?hook_data ~ubind ~impargs ce =
   end;
   dref
 
-let declare_mutual ~opaque ~univs ~scope ~kind ~ubind fixnames fixdecls fixtypes fiximps =
-  CList.map4
-    (fun name body types impargs ->
-       let ce = Declare.definition_entry ~opaque ~types ~univs body in
-       declare_definition ~name ~scope ~kind ~ubind ~impargs ce)
-    fixnames fixdecls fixtypes fiximps
+let declare_mutual ~cofix ~indexes ~opaque ~univs ~scope ~kind ~ubind fixnames fixdecls fixtypes fiximps =
+  let csts = CList.map4
+      (fun name body types impargs ->
+         let ce = Declare.definition_entry ~opaque ~types ~univs body in
+         declare_definition ~name ~scope ~kind ~ubind ~impargs ce)
+      fixnames fixdecls fixtypes fiximps
+  in
+  Declare.recursive_message (not cofix) indexes fixnames;
```

Comment:
I think I'd prefer to take `isfix` and negate in the caller.

Reply:
Looking at later commits `cofix` becomes derived from `possible_indexes` so it doesn't really matter.

Reply:
Yup, I'm trying to do my best here but indeed there is some churn in the API until we arrive to a stable thing.

Reply:
solved.